src/UI/Movies/Editor/MovieEditorFooterView.js

```diff
@@ -36,13 +37,32 @@ module.exports = Marionette.ItemView.extend({
 
     initialize : function(options) {
         this.moviesCollection = options.collection;
+		this.fullMovieCollection = options.c2;
 
         RootFolders.fetch().done(function() {
             RootFolders.synced = true;
         });
 
         this.editorGrid = options.editorGrid;
-        this.listenTo(this.moviesCollection, 'backgrid:selected', this._updateInfo);
+        //this.listenTo(this.moviesCollection, 'backgrid:selected', this._updateInfo);
+
+
+        this.listenTo(this.moviesCollection, 'backgrid:selected', function(model, selected) {
+            //window.alert(selected);
+            var m =  this.fullMovieCollection.findWhere({ tmdbId : parseInt(model.get('tmdbId'),10) });
```

Comment:
@fedoranimus : im still working on it quite a bit.. i have a much better and cleaner implementation coming in the next commit.. still working on it.. dont bother testing until I let you know...

I'll look into your other comments later..